plugin.video.nrk/addon.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="plugin.video.nrk"
+       name="NRK Nett-TV"
+       version="4.8.0"
+       provider-name="takoi+a99b">
+  <requires>
+    <import addon="xbmc.python" version="3.0.1"/>
+    <import addon="script.module.requests" version="1.0.4"/>
+    <import addon="script.module.routing" version="0.1.0"/>
+  </requires>
+  <extension point="xbmc.python.pluginsource" library="main.py">
+    <provides>video</provides>
+  </extension>
+  <extension point="xbmc.addon.metadata">
+    <platform>all</platform>
+    <language>no</language>
+    <summary lang="en">Watch NRK Web TV</summary>
+    <summary lang="no">Se NRK Nett-TV</summary>
+    <summary lang="it">Guarda NRK (in Norvegia)</summary>
+    <description lang="en">NRK Web TV gives you the opportunity to watch live TV as well as recordings of many shows.</description>
+    <description lang="no">NRK Nett-TV gir deg muligheten til å se mange av NRKs programmer både direkte og i opptak.</description>
+    <description lang="it">Questo addon per NRK rende possibile la visione di molti programmi di NRK sia in diretta che in differita.</description>
+    <license>GNU GENERAL PUBLIC LICENSE. Version 3, 29 June 2007</license>
```

Comment:
Could you please use the corresponding SPDX license identifier: https://spdx.org/licenses/

Reply:
no problem, done